common/scala/src/main/scala/org/apache/openwhisk/common/Logging.scala

```diff
@@ -330,14 +330,14 @@ object LoggingMarkers {
   val CONTROLLER_ACTIVATION_BLOCKING_DATABASE_RETRIEVAL =
     LogMarkerToken(controller, "blockingActivationDatabaseRetrieval", count)
 
-  // Time that is needed load balance the activation
+  // Time that is needed to load balance the activation
   val CONTROLLER_LOADBALANCER = LogMarkerToken(controller, loadbalancer, start)
 
   // Time that is needed to produce message in kafka
   val CONTROLLER_KAFKA = LogMarkerToken(controller, kafka, start)
 
   // System overload and random invoker assignment
-  val SYSTEM_OVERLOAD = LogMarkerToken(controller, "systemOverload", count)
+  def SYSTEM_OVERLOAD(actionType: String) = LogMarkerToken(controller, s"systemOverload$actionType", count)
```

Comment:
Any performance concerns about changing this from a val field to a function that will do a string interpolation on every invocation?

Reply:
Agreed, let's make two static values SYSTEM_OVERLOAD_MANAGED and SYSTEM_OVERLOAD_BLACKBOX to get rid of that work on each invocation.